src/components/aws/CloudFront.ts

```diff
@@ -23,7 +23,7 @@ export class CloudFront extends pulumi.ComponentResource {
 
   public readonly targetDomain: pulumi.Output<string>
 
-  constructor (
+  constructor(
```

Comment:
Ne respecte pas le linting du projet

```suggestion
  constructor (
```